src/Execution/ModelsLoader/PaginatedModelsLoader.php

```diff
@@ -116,6 +120,10 @@ protected function relationInstance(EloquentCollection $parents): Relation
 
     /**
      * Return a fresh instance of a query builder for the underlying model.
+     *
+     * @param EloquentCollection<int, Model> $parents
```

Comment:
```suggestion
     * @param  EloquentCollection<int, Model>  $parents
```

Always use two spaces for `@param`